src/scheduler.cpp

```diff
@@ -111,7 +111,8 @@ static void Repeat(CScheduler& s, CScheduler::Function f, std::chrono::milliseco
 
 void CScheduler::scheduleEvery(CScheduler::Function f, std::chrono::milliseconds delta)
 {
-    scheduleFromNow([=] { Repeat(*this, f, delta); }, delta);
+    auto* self{this}; // Avoid warning: implicit capture of ‘this’ via ‘[=]’ is deprecated in C++20 [-Wdeprecated]
```

Comment:
Why not just make it explicit?

Reply:
Because it wouldn't compile with C++17:

```
    [=, this] {};   // until C++20: Error: this when = is the default
```

See also https://en.cppreference.com/w/cpp/language/lambda#Lambda_capture

Reply:
Being fully explicit should work? `[this, f, delta] { Repeat(*this, f, delta; }`